apps/frontend/src/libs/components/header/libs/components/notifications-popover/notifications-popover.tsx

```diff
@@ -18,26 +21,46 @@ type Properties = {
 	onClose: () => void;
 };
 
+const PAGE_SIZE = 10;
```

Comment:
should be in a separate file

Reply:
This has since been adjust, ty for the review